src/sentry/static/sentry/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -198,11 +198,19 @@ class AbstractIntegrationDetailedView<
     return {organization, features};
   }
 
+  cleanTags(featureData) {
```

Comment:
@ceorourke I would recommend making this function take no inputs than just calling `this.featureData`. Otherwise, you should add a type to the input of this function because right now it is being typed as `any`.